config/migrations/2024/20240812111040-feat-add-merged-municipalities-and-ocmws/20240821122839-merged-municipality-beveren_kruibeke_zwijndrecht.sparql

```diff
@@ -0,0 +1,251 @@
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX org: <http://www.w3.org/ns/org#>
+PREFIX organisatie: <https://data.vlaanderen.be/ns/organisatie#>
+PREFIX regorg: <http://www.w3.org/ns/regorg#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX adms: <http://www.w3.org/ns/adms#>
+PREFIX generiek: <https://data.vlaanderen.be/ns/generiek#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX prov: <http://www.w3.org/ns/prov#>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX geo: <http://www.opengis.net/ont/geosparql#>
+
+INSERT {
```

Comment:
This insert has to be `INSERT DATA`. I think the spec does not allow a lone INSERT.

The other `xxx-merged-yyy.sparql` also have the same issue in the same location; I'll add a quick comment on each one.

Reply:
Interestingly a line `INSERT` does work, but you are right that an explicit `INSERT DATA` is better here.

Note: I regenerated the migrations so the filenames (and UUIDs) changed in the process.

Reply:
Yes indeed, it seems `virtuoso` accepts the lone insert (probably transforms it to an insert data internally).